.travis.yml

```diff
@@ -51,42 +51,27 @@ script:
   - moodle-plugin-ci validate
   - moodle-plugin-ci savepoints
   - moodle-plugin-ci mustache
-  - moodle-plugin-ci grunt || [ "$MOODLE_BRANCH" != 'master' -a "$MOODLE_BRANCH" != 'MOODLE_38_STABLE' ]
+  - moodle-plugin-ci grunt || [ "$MOODLE_BRANCH" != 'master' -a "$MOODLE_BRANCH" != 'MOODLE_39_STABLE' ]
```

Comment:
Ok, so... the fixture has been modified to match what we are getting now with 39_STABLE and master. And, for the rest of branches (35, 36, 37, 38), we exceptionally allow them to pass, because the "compiled" js is different.

So far, it's ok for me. I'm going to resolve this and proceed. If we find some better way to cover those different results, great, but in any case, to have covered the actual ones is better than the old ones, so all right. IMO.

Thanks!